instruqt/proxy/config.yml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+virtualmachines:
+- name: proxy-host
+  image: centos-cloud/centos-7
+  shell: /bin/bash
+  machine_type: n1-standard-1
+  allow_external_ingress:
+  - http
+  - https
+  - high-ports
+- name: isolated-host
+  image: kots-field-labs/instruqt-airgap
```

Comment:
Oh, we chatted about this when I needed a bigger instance size for kurl.  What is that image does it have the larger disk? (for the PV) happy to change if it will just work?  or do some post release update testing?  Your pov image works great but appreciate that it's a bit old now and maybe has some bloat in there.. or we want to remove it